custom/icds_reports/utils/aggregation_helpers/distributed/ccs_record_monthly.py

```diff
@@ -90,8 +90,7 @@ def aggregation_query(self):
         valid_in_month = "( {} OR {})".format(pregnant_to_consider, lactating)
 
         add_in_month = "(case_list.add>= {} AND case_list.add<={})".format(start_month_string, end_month_string)
-        add_in_month_after_opened_on = "({} AND case_list.opened_on<=case_list.add)".format(add_in_month)
-        delivered_in_month = "({} AND {})".format(seeking_services, add_in_month_after_opened_on)
+        delivered_in_month = "({} AND {} AND agg_delivery.case_id IS NOT NULL)".format(seeking_services, add_in_month)
```

Comment:
i'm assuming we are deliberately not writing code that isn't prone to sql injection here because we know there is no chance of user input on these values and this makes things easier for us to maintain.

if that is true and agreed on, it could be a good opportunity to use f-strings to make things more readable :)